lib/uwb/include/uwb/protocols/fira/FiraDevice.hxx

```diff
@@ -549,7 +549,7 @@ using UwbApplicationConfigurationParameterValue = std::variant<
     RangingRoundUsage, // RANGING_ROUND_USAGE, tag0x01, size 1
     RangingMode, // RANGING_TIME_STRUCT, tag 0x1A, size 1
     RangingRoundControl, // RANGING_ROUND_CONTROL, tag 0x0C, size 1,
-    ResultReportConfiguration, // RESULT_REPORT_CONFIG, tag 0x2E, size 1
+    ResultReportConfiguration, // RESULT_REPORT_CONFIG, tag 0x2E, size 1 // TODO this should really be a set
```

Comment:
Might as well fix this now so we can close this chapter.

Reply:
I would prefer to do this in another PR (keeps this one small)

Reply:
👍